src/App.js

```diff
@@ -23,49 +23,108 @@ const generateSquares = () => {
   }
 
   return squares;
-}
+};
 
 const App = () => {
-
+  console.log(generateSquares());
```

Comment:
When submitting code to be reviewed, we should eliminate print statements/console logs

```suggestion
```